src/Storages/StorageBuffer.h

```diff
@@ -102,7 +108,7 @@ friend class BufferBlockOutputStream;
 
 
 private:
-    Context global_context;
```

Comment:
It will be great to use forward declaration for the Context in the similar places

Reply:
Yeah. It'll be more consistent. Done.